src/styles/index.ts

```diff
@@ -4476,10 +4476,8 @@ const styles = (theme: ThemeColors) =>
             width: '100%',
         },
         menuItemError: {
-            position: 'absolute',
-            bottom: -4,
-            left: 20,
-            right: 20,
+            marginTop: 4,
+            marginBottom: 0,
         },
```

Comment:
Looks like this may have caused this error: https://github.com/Expensify/App/issues/42209